sdk/identity/Azure.Identity/src/KnownAuthorityHosts.cs

```diff
@@ -10,24 +10,50 @@ namespace Azure.Identity
     /// </summary>
     public static class KnownAuthorityHosts
     {
+        private const string AzureCloudHostUrl = "https://login.microsoftonline.com/";
+        private const string AzureChinaCloudHostUrl = "https://login.chinacloudapi.cn/";
+        private const string AzureGermanCloudHostUrl = "https://login.microsoftonline.de/";
+        private const string AzureUSGovernmentHostUrl = "https://login.microsoftonline.us/";
         /// <summary>
         /// The host of the Azure Active Directory authority for tenants in the Azure Public Cloud.
         /// </summary>
-        public static readonly Uri AzureCloud = new Uri("https://login.microsoftonline.com/");
+        public static readonly Uri AzureCloud = new Uri(AzureCloudHostUrl);
 
         /// <summary>
         /// The host of the Azure Active Directory authority for tenants in the Azure China Cloud.
         /// </summary>
-        public static readonly Uri AzureChinaCloud = new Uri("https://login.chinacloudapi.cn/");
+        public static readonly Uri AzureChinaCloud = new Uri(AzureChinaCloudHostUrl);
 
         /// <summary>
         /// The host of the Azure Active Directory authority for tenants in the Azure German Cloud.
         /// </summary>
-        public static readonly Uri AzureGermanCloud = new Uri("https://login.microsoftonline.de/");
+        public static readonly Uri AzureGermanCloud = new Uri(AzureGermanCloudHostUrl);
 
         /// <summary>
         /// The host of the Azure Active Directory authority for tenants in the Azure US Government Cloud.
         /// </summary>
-        public static readonly Uri AzureUSGovernment = new Uri("https://login.microsoftonline.us/");
+        public static readonly Uri AzureUSGovernment = new Uri(AzureUSGovernmentHostUrl);
```

Comment:
Should this end in `Cloud` to mathc the others?

Reply:
Same as above.